mp-istio-sample/serviceA/src/main/java/application/rest/ServiceAEndpoint.java

```diff
@@ -0,0 +1,86 @@
+package application.rest;
+
+import java.util.Date;
+
+import javax.enterprise.context.RequestScoped;
+import javax.inject.Inject;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.client.ClientBuilder;
+import javax.ws.rs.core.Application;
+import javax.ws.rs.core.MediaType;
+
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+import org.eclipse.microprofile.faulttolerance.Fallback;
+import org.eclipse.microprofile.faulttolerance.Retry;
+
+@RequestScoped
+@Path("serviceA")
+public class ServiceAEndpoint {
+
+    @Inject
+    @ConfigProperty(name = "svcBHost", defaultValue = "localhost")
+    private String serviceBHost;
+
+    @Inject
+    @ConfigProperty(name = "svcBPort", defaultValue = "9080")
+    private String serviceBPort;
+
+    StringBuilder url;
+    static int callCount;
+    int tries;
+
+    @GET
+    @Retry
+    @Fallback(fallbackMethod="serviceAFallback")
+    @Produces(MediaType.TEXT_PLAIN)
+    public String callServiceB() {
+
+      ++callCount;
+      ++tries;
+
+      url = new StringBuilder();
+      url.append("http://")

```

Comment:
I appreciate that this is a way to make it runnable on localhost or in the cloud, but I'd prefer it was specific to cloud and required a local cloud, like minikube, to run than using localhost everywhere.

Especially as it shouldn't be necessary to use ports when calling services in Kubernetes. It's showcasing not ideal practices with this approach